src/test/java/com/olgaboiar/mint/ResponseTest.java

```diff
@@ -0,0 +1,49 @@
+package com.olgaboiar.mint;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ResponseTest {
+    String currentDate;
+    String date;
+    Header header;
+
+    @BeforeEach
+    public void init(){
+        currentDate = DateTimeFormatter.RFC_1123_DATE_TIME.format(ZonedDateTime.now());
+        date = "Date: " + currentDate;
+        header = new Header("200 OK", "Content-Type: text/html", date);
+    }
+
+    @Test
+    void responseHasBlankLineWhenBodyIsAdded() {
+        Body body = new Body("body");
+        Response testResponse = new Response (header, body);
+        String actual = testResponse.prepareResponse();
+        String expected = "200 OK\n" +
+                "Content-Type: text/html\n" +
+                "Date: " + currentDate + "\n" +
+                "\n" +
+                "body";
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void responseHasNoBlankLineWithoutBody() {
```

Comment:
It seems to me that we're testing whether it returns a valid body? I think that's more relevant to what we're trying to test here. At a high-level point of view, we only care about its behavior which is returning a valid response. We don't really care about how it's done.

Reply:
added tests in https://github.com/olgaboiar/MintHTTPServer/pull/10/commits/69a33d0448dccb8621923c293fbbf8f255b85856